docs/sources/operators-guide/configuring/configuring-tsdb-block-upload.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Configuring TSDB block upload"
+menuTitle: "Configuring TSDB block upload"
+description: "Learn how to configure Grafana Mimir to enable TSDB block upload"
+weight: 120
+---
+
+# Configuring TSDB block upload
+
+Grafana Mimir supports uploading of historic TSDB blocks, sourced from f.ex. Prometheus, Thanos, or
```

Comment:
It's tricky to mention Thanos because of the current limitations related to external labels (see the work Peter is doing in https://github.com/grafana/mimir/pull/2122). It's fine mentioning Thanos, but we should link `docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md` to learn more about known limitations.

Reply:
Thanks, I added a note regarding this.